sdk-core/src/main/java/care/data4life/sdk/fhir/FhirContract.kt

```diff
@@ -19,7 +19,7 @@ package care.data4life.sdk.fhir
 import care.data4life.crypto.GCKey
 
 //TODO: make internal
-interface FhirContract {
+class FhirContract {
```

Comment:
why is this changed to a class?

Reply:
You right, forgot to change it back - originally I wanted the enum to `internal`, which currently does not work. However as we have it in another place - interfaces do not support those kind of visibility modifiers.